### ISSUE TITLE
Publish some newer portfolio showcases on the home page

### ISSUE THREAD

**Issue by brashrebel** [realbig/RBM-Theme#37]
Peaceful Dragon School and VDK? Really? That's the best we've got to show? PDS is using a stock, barely modified WooTheme and VDK is super meh.

Inosencio, Pat Robertson, Mid Michigan Health Center, Impact Insurance, ALS, MSP Games, Project Freedom, shoot even Epic! All those clients got a _custom_ site designed by us. Some were prettier than others of course but we ultimately executed a completely unique design. Pick from these and make our showcase great again.

**Comment by joelworsham** [comment]
VDK is only mildly meh thank you! (I still kind of like it)

**Comment by joelworsham** [comment]
Also MSP Games is gone-dizzle I believe.

... but yes, much better options ;)

**Comment by BigActual** [comment]
I like vdk as well.  I'll put this on my to do list.

**Comment by BigActual** [comment]
Of i never hear make it great again it'll be too soon.

**Comment by BigActual** [comment]
Need URLs for Pat Robertson, Need clarification on MSP Games. Likely won't use Epic because it's been so badly damaged by changes

**Comment by brashrebel** [comment]
http://patriciamrobertson.com/
MSP Games exists no longer.

And we don't need _all_ of them in the home page showcase. Just the ones that truly look best in that little graphic with the three devices. I'd say our top four would suffice.

**Comment by BigActual** [comment]
Yeah, thanks for the domain. Sorry I had to ask, but it's hard to google your way around Evangelical Pat Robertson
